Groupon UP/Groupon UP/UPViewController.swift

```diff
@@ -229,8 +228,8 @@ extension UPViewController {
             v.layer.borderWidth = UPBorderWidth
             v.layer.borderColor = UIColor.lightGrayColor().CGColor
             v.layer.cornerRadius = UPBorderRadius
-            if up != nil {
-               v.text = up.message
+            if self.deal.up != nil {
```

Comment:
We've check the validity of self.deal.up already, so we can remove this step. Another option is to `let up = self.deal.up` in line 123